### ISSUE TITLE
science goggles feature request

### ISSUE THREAD

**Issue by pedr69** [vgstation-coders/vgstation13#35185]
[featurerequest]
what if they were made better than the current "completely useless"

my suggestion is giving them a VERY primitive view variables
so if you examine the HOS' hat you get its armor values
recyclable items have their material contents listed
the floor mounted scrubbers show their siphoning speeds on scrub and siphon modes
maybe add an overlay to items with high tech levels so they glow and stand out in the purple world of the scigger goggle

basically give it a bunch of fucking tooltips and HUDs and random shit
all this would be a lot of work so i'm making a feature request instead of coding it

**Comment by ValkyrieSkies** [comment]
Science Goggles aren't actually completely useless though, when turned on they let you view pathogen spread which is super useful for medbay and janitors. The description is just very outdated.

**Comment by pedr69** [comment]
yeah that's the only thing they do and im wondering if people would like them to do more or not before i spend 10 hours coding it
why don't the doctors healthHUDS get the pathogen overlay anyway(the CMO advanced HUD does)

**Comment by Eneocho** [comment]
>why don't the doctors healthHUDS get the pathogen overlay anyway

Muh balance, that's why. It's perfectly reasonable that the people that mainly deal with diseases and shit would have ~~nightmare vision goggles~~ pathogen goggles, but it's gated to CMOs and nerds cause muh balance with diseases and virologists.

**Comment by ValkyrieSkies** [comment]
I kinda like Pedr's suggestion though - goggles that let you see an item's tech levels and recycling materials actually sound like something that'd be pretty neat for science to have. Honestly maybe the existing science goggles should be renamed something else now given that they're more of a medbay item now and new science goggles be added with the proposed effect?

**Comment by SonixApache** [comment]
the goggles do nothing

also you can wear them over your medHUDs

**Comment by Eneocho** [comment]
Requesting the science goggles get renamed to "Nightmare Vision Goggles"
